database/models/tables.py

```diff
@@ -36,6 +36,24 @@ def add_new_artist(self):
         insert = insert_statement("artist", [], [])
         self.execute_query(insert)
 
+    def get_single_artist_info(self, artist_name):
```

Comment:
I think get_single_artist_info and get_albums_for_artist could probably be combined into one query with an inner join. Album entities have an artist_id attribute so I thiiiiink you should be able to do `inner join album on artist.id=album.artist_id`

Reply:
Note: I don't consider this something that _needs_ to change. It seems to be working as it is; however, it might be more efficient do an inner join than to run two queries.